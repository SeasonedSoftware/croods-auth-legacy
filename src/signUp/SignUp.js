import React, { cloneElement } from 'react'
import New from '../New'

import Created from '../Created'

export default props => {
  const { relativePath = '/auth' } = props

  return <New {...props} name="auth.signUp" path={relativePath} />
}
