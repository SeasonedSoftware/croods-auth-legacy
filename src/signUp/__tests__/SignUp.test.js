import React from 'react'
import renderer from 'react-test-renderer'

import SignUp from '../SignUp'

jest.mock('../../New', () => props => <div {...props}>New</div>)

it('renders correctly', () => {
  const props = { foo: 'bar' }
  const tree = renderer.create(<SignUp {...props} />).toJSON()

  expect(tree).toMatchSnapshot()
})
