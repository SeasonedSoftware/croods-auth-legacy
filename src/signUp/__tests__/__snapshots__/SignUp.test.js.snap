// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  foo="bar"
  name="auth.signUp"
  path="/auth"
>
  New
</div>
`;
